### PR TITLE
Adding Common folder and some smaller refactorings

### PR DESCRIPTION
**Pull request by matzew**
* moved duplicated code to `common` module
* removal of `printStackTrace` and use logger, where possible
* removed `syso` to use logger
* misc

### CODE REVIEW

File: manager/src/main/java/io/streamzi/openshift/API.java [hunk 1]
```diff
@@ -154,14 +153,14 @@ public void postFlow(String flowJson) {
     public String getGlobalProperties() {
         final Properties props = new Properties();
 
-        String bootstrapServers = EnvironmentResolver.get(ProcessorConstants.KAFKA_BOOTSTRAP_SERVERS);
+        final String bootstrapServers = EnvironmentResolver.get(ProcessorConstants.KAFKA_BOOTSTRAP_SERVERS);
```
Comment (sjwoodman) [inline manager/src/main/java/io/streamzi/openshift/API.java]: These are no longer used and can be removed. I've added a task to the spreadsheet to do this.

Comment (matzew) [reply manager/src/main/java/io/streamzi/openshift/API.java]: I will assign that task to me :-)

Comment (sjwoodman) [reply manager/src/main/java/io/streamzi/openshift/API.java]: Assigned... :)

Comment (matzew) [reply manager/src/main/java/io/streamzi/openshift/API.java]: ok, to keep things clean, that's a different PR

**Pull request merged**